benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadServer.java

```diff
@@ -141,7 +142,7 @@
     if (config.getServerType() == Control.ServerType.ASYNC_GENERIC_SERVER) {
       serverBuilder.addService(
           ServerServiceDefinition
-              .builder(BenchmarkServiceGrpc.SERVICE_NAME)
+              .builder(new ServiceDescriptor(BenchmarkServiceGrpc.SERVICE_NAME))
```

Comment:
If the service implements a method that isn't defined in the ServiceDescriptor, then "you're going to have a bad time." I can't think of any legitimate need for that and I do think it could find bugs. Yes, it will add a few lines to some tests, but it didn't seem it would be too onerous.

If we're going to do a warning then it seems we might as well do an exception. I have a hard time believing that something that produced inconsistent objects would only sometimes fail. Failing with an exception seems easiest to find the issue without being dangerous and won't be prone to accumulating logspam.